orttraining/orttraining/training_ops/cpu/controlflow/wait.h

```diff
@@ -2,15 +2,16 @@
 // Licensed under the MIT License.
 
 #pragma once
-#include <thread>
-#include <chrono>
 #include "core/common/common.h"
 #include "core/framework/op_kernel.h"
 #include "event_pool.h"
 
 namespace onnxruntime {
 namespace contrib {
 
+// Wait for the event ID stored in the input tensor.
+void wait_event_in_tensor(const Tensor* event_id_tensor);
```

Comment:
pass by const ref, if event_id_tensor cannot be optional or missing

Reply:
Ok.